tests/test_transforms/test_math/test_binary_operator_transform.py

```diff
@@ -0,0 +1,339 @@
+        ("-", "target", "feature", "target"),
+        ("*", "target", "feature", "target"),
+        ("/", "target", "feature", "target"),
+        ("/=", "target", "feature", "target"),
+        ("%", "target", "feature", "target"),
+        ("**", "target", "feature", "target"),
+        ("==", "target", "feature", "target"),
+        (">=", "target", "feature", "target"),
+        ("<=", "target", "feature", "target"),
+        (">", "target", "feature", "target"),
+        ("<", "target", "feature", "target"),
+        ("+", "feature", "target", "feature"),
+        ("-", "feature", "target", "feature"),
+        ("*", "feature", "target", "feature"),
+        ("/", "feature", "target", "feature"),
+        ("/=", "feature", "target", "feature"),
+        ("%", "feature", "target", "feature"),
+        ("**", "feature", "target", "feature"),
+        ("==", "feature", "target", "feature"),
+        (">=", "feature", "target", "feature"),
+        ("<=", "feature", "target", "feature"),
+        (">", "feature", "target", "feature"),
+        ("<", "feature", "target", "feature"),
+        ("+", "target", "feature", "feature"),
+        ("-", "target", "feature", "feature"),
+        ("*", "target", "feature", "feature"),
+        ("/", "target", "feature", "feature"),
+        ("/=", "target", "feature", "feature"),
+        ("%", "target", "feature", "feature"),
+        ("**", "target", "feature", "feature"),
+        ("==", "target", "feature", "feature"),
+        (">=", "target", "feature", "feature"),
+        ("<=", "target", "feature", "feature"),
+        (">", "target", "feature", "feature"),
+        ("<", "target", "feature", "feature"),
+        ("+", "feature", "target", "new_col"),
+        ("-", "feature", "target", "new_col"),
+        ("*", "feature", "target", "new_col"),
+        ("/", "feature", "target", "new_col"),
+        ("/=", "feature", "target", "new_col"),
+        ("%", "feature", "target", "new_col"),
+        ("**", "feature", "target", "new_col"),
+        ("==", "feature", "target", "new_col"),
+        (">=", "feature", "target", "new_col"),
+        ("<=", "feature", "target", "new_col"),
+        (">", "feature", "target", "new_col"),
+        ("<", "feature", "target", "new_col"),
+        ("+", "target", "feature", "new_col"),
+        ("-", "target", "feature", "new_col"),
+        ("*", "target", "feature", "new_col"),
+        ("/", "target", "feature", "new_col"),
+        ("/=", "target", "feature", "new_col"),
+        ("%", "target", "feature", "new_col"),
+        ("**", "target", "feature", "new_col"),
+        ("==", "target", "feature", "new_col"),
+        (">=", "target", "feature", "new_col"),
+        ("<=", "target", "feature", "new_col"),
+        (">", "target", "feature", "new_col"),
+        ("<", "target", "feature", "new_col"),
+    ],
+)
+def test_simple_one_segment(ts_one_segment: TSDataset, operand, left_column, right_column, out_column):
```

Comment:
I doesn't really understand why should we test one and two segments separately. It is the same transform that works with them.

Reply:
Yasha asked to make different tests in order to check that it works correctly